Pod/ILGClasses/ILGClasses.h

```diff
@@ -8,6 +8,8 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I'm not saying it has to go.  I'm just saying eww.  At least, until I think about it some more.